cargo-apk/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
```

Comment:
Might want to keep this around for the next release... I accidentally removed it in one of the previous releases too and it botched some rebases.